### PR TITLE
#1551 Meters support in switch validation.

### PR DESCRIPTION
**Pull request by dpoltavets**
- added new API path `/switches/{switch_id}/validate`;
- added switch meter validation.

Closes: #1551.

---
This change is [Reviewable](https://reviewable.io/reviews/telstra/open-kilda/2066)

### CODE REVIEW

File: services/src/messaging/src/main/java/org/openkilda/messaging/info/switches/MeterMisconfiguredInfoEntry.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    private Long rate;
+
+    @JsonProperty("burst_size")
+    private Long burstSize;
+
+    @JsonProperty("flags")
+    private String[] flags;
+
+    public MeterMisconfiguredInfoEntry() {
+    }
```
Comment (niksv) [inline services/src/messaging/src/main/java/org/openkilda/messaging/info/switches/MeterMisconfiguredInfoEntry.java]: you can use @NoArgsConstructor

File: services/wfm/src/main/java/org/openkilda/wfm/topology/switchmanager/SwitchManagerTopologyConfig.java [hunk 2]
```diff
@@ -30,4 +36,19 @@ default String getKafkaNorthboundTopic() {
     default String getKafkaSpeakerTopic() {
         return getKafkaTopics().getSpeakerTopic();
     }
+
+    @Key("flow.meter.burst.coefficient")
```
Comment (niksv) [inline services/wfm/src/main/java/org/openkilda/wfm/topology/switchmanager/SwitchManagerTopologyConfig.java]: there is no such option. we have only `burst.coefficient`

Comment (dpoltavets) [reply services/wfm/src/main/java/org/openkilda/wfm/topology/switchmanager/SwitchManagerTopologyConfig.java]: Fixed.

File: services/src/api/src/main/java/org/openkilda/northbound/dto/switches/RulesSyncResult.java [hunk 3]
```diff
@@ -27,14 +27,14 @@
 @NoArgsConstructor
 public class RulesSyncResult extends RulesValidationResult {
```
Comment (sergii-iakovenko) [inline services/src/api/src/main/java/org/openkilda/northbound/dto/switches/RulesSyncResult.java]: We must not amend the existing API!

File: services/src/api/src/main/java/org/openkilda/northbound/dto/switches/RulesValidationResult.java [hunk 4]
```diff
@@ -27,12 +27,12 @@
 @NoArgsConstructor
 public class RulesValidationResult {
```
Comment (sergii-iakovenko) [inline services/src/api/src/main/java/org/openkilda/northbound/dto/switches/RulesValidationResult.java]: We must not amend the existing API!

File: services/src/api/src/main/java/org/openkilda/northbound/dto/switches/RulesSyncResult.java (hunk 3 above)
Comment (dpoltavets) [reply services/src/api/src/main/java/org/openkilda/northbound/dto/switches/RulesSyncResult.java]: Fixed.

File: services/src/api/src/main/java/org/openkilda/northbound/dto/switches/RulesValidationResult.java (hunk 4 above)
Comment (dpoltavets) [reply services/src/api/src/main/java/org/openkilda/northbound/dto/switches/RulesValidationResult.java]: Fixed.

**Pull request merged**